src/bindgen/library.rs

```diff
@@ -758,6 +797,7 @@ pub struct GeneratedBindings {
     monomorphs: Monomorphs,
     items: Vec<PathValue>,
     functions: Vec<Function>,
+    full_objects: Vec<Struct>,
```

Comment:
Can you rename this to `member_function_structs`? I don't mind the longer name as long as it is clear.